### PR TITLE
21w20a entity

### PR DESCRIPTION
**Pull request by apple502j**
Mapped new 21w20a entity methods and stuff.

### CODE REVIEW

File: mappings/net/minecraft/entity/passive/PufferfishEntity.mapping [hunk 1]
```diff
@@ -2,10 +2,13 @@ CLASS net/minecraft/class_1454 net/minecraft/entity/passive/PufferfishEntity
 	FIELD field_30353 NOT_PUFFED I
 	FIELD field_30354 SEMI_PUFFED I
 	FIELD field_30355 FULLY_PUFFED I
+	FIELD field_33692 BLOW_UP_PREDICATE Lnet/minecraft/class_4051;
```
Comment (haykam821) [inline mappings/net/minecraft/entity/passive/PufferfishEntity.mapping]: How does this differ from `BLOW_UP_FILTER`?

Relates to #1624

File: mappings/net/minecraft/entity/LivingEntity.mapping [hunk 2]
```diff
@@ -265,7 +265,11 @@ CLASS net/minecraft/class_1309 net/minecraft/entity/LivingEntity
 	METHOD method_35054 setNoDrag (Z)V
 		ARG 1 noDrag
 	METHOD method_36362 updateGlowing ()V
+	METHOD method_36549 addStingerParticles ()V
```
Comment (Shnupbups) [inline mappings/net/minecraft/entity/LivingEntity.mapping]: This is completely incorrect, the particles have nothing to do with stingers and are in fact the 'poof' particles that result from the death of the entity. In Enigma the 'stingers' field is mistakenly shown as the value being iterated over in the for loop as a result of a bug.

Comment (Shnupbups) [reply mappings/net/minecraft/entity/LivingEntity.mapping]: Mapped correctly in #2417

File: mappings/net/minecraft/entity/passive/PufferfishEntity.mapping (hunk 1 above)
Comment (liach) [reply mappings/net/minecraft/entity/passive/PufferfishEntity.mapping]: guess this is a target predicate?

File: mappings/net/minecraft/entity/LivingEntity.mapping (hunk 2 above)
Comment (liach) [reply mappings/net/minecraft/entity/LivingEntity.mapping]: Yeah, sorry, but enigma's cfr appears to incorrectly reuse local variables. If you see something is a local variable that is not in the list of parameters, then for sure enigma's cfr broke there

File: mappings/net/minecraft/entity/player/PlayerEntity.mapping [hunk 3]
```diff
@@ -2,6 +2,7 @@ CLASS net/minecraft/class_1657 net/minecraft/entity/player/PlayerEntity
 	FIELD field_18134 POSE_DIMENSIONS Ljava/util/Map;
 	FIELD field_18135 STANDING_DIMENSIONS Lnet/minecraft/class_4048;
 	FIELD field_19428 shoulderEntityAddedTime J
+	FIELD field_30642 OFFLINE_PLAYER_PREFIX Ljava/lang/String;
```
Comment (liach) [inline mappings/net/minecraft/entity/player/PlayerEntity.mapping]:
```suggestion
	FIELD field_30642 OFFLINE_PLAYER_UUID_PREFIX Ljava/lang/String;
```
as this is prepended to an offline player's nickname for uuid generation

File: mappings/net/minecraft/entity/passive/PufferfishEntity.mapping (hunk 1 above)
Comment (liach) [reply mappings/net/minecraft/entity/passive/PufferfishEntity.mapping]:
```suggestion
	FIELD field_33692 BLOW_UP_TARGET_PREDICATE Lnet/minecraft/class_4051;
```

File: mappings/net/minecraft/entity/LivingEntity.mapping (hunk 2 above)
Comment (liach) [reply mappings/net/minecraft/entity/LivingEntity.mapping]:
```suggestion
```

**Review by liach — APPROVED** [review]

**Review by enbrain — APPROVED** [review]

**Pull request merged**